bin/run-db-download.py

```diff
@@ -18,13 +19,26 @@
     set_db_data,
 )
 
+# ROOT path of the project. A pathlib.Path object.
+ROOT_PATH = Path(__file__).resolve().parents[1]
+ROOT = str(ROOT_PATH)
+
+# add bedrock to path
+sys.path.append(ROOT)
+
+# must import after adding bedrock to path
+from bedrock.base.config_manager import config  # noqa
+
 BUCKET_NAME = os.getenv("AWS_DB_S3_BUCKET", "bedrock-db-dev")
 REGION_NAME = os.getenv("AWS_DB_REGION", "us-west-2")
```

Comment:
Since we have `config` now, we could optionally convert these (or anywhere else in this file where we use `os.getenv`). An example of a string value would just be:

```
BUCKET_NAME = config("AWS_DB_S3_BUCKET", default="bedrock-db-dev")
```

Sorry to tack on so much extra.

Reply:
No problem at all!